modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/servlet/taglib/JSLoaderTopHeadDynamicInclude.java

```diff
@@ -131,7 +131,7 @@ private String _getResolvePath(HttpServletRequest httpServletRequest) {
 				httpServletRequest);
 
 		return absolutePortalURLBuilder.forServlet(
-			_jsResolveModulesServlet.getURL()
+			JSLoaderModulesUtil.getURL()
```

Comment:
@MrJohn1911 @rafaprax You may have a null value here.

Before your change, it is using _jsResolveModulesServlet to retrieve the url, and in the activate method of _jsResolveModulesServlet, the url is setup.

After your change, you won't be sure when the url is populated.